pkgs/tools/system/bpytop/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, python3Packages, fetchFromGitHub, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "bpytop";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "aristocratos";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0cxyrk5a9j0ymll9h5b6jq48yjy9srcxh4rmsqk8w0d14prmflgg";
+  };
+
+  phases = [ "unpackPhase" "installPhase" ];
+
+  buildInputs = [ python3Packages.python makeWrapper ];
+  propagatedBuildInputs = [ python3Packages.psutil ];
+
+  installPhase = ''
+    sed -i -e "s#/usr/\[local/\]#$out/#g" \
```

Comment:
Should this happen in `fixupPhase`?

Reply:
Fixed that.  I did it in `installPhase` since I overwrite the default phases because of the non-standard install procedure, but yes, this should be the nicer way.